resources/config-defaults.edn

```diff
@@ -137,15 +137,8 @@
                        :remove nil}
 
  ;; External systems for pushing entitlements (v2)
- ;; NB: remember feature flag `:enable-ega`
  ;;
- ;; Example:
- ;; :entitlement-push [{:id "ega"
- ;;                     :type :ega
- ;;                     :connect-server-url "https://ega.ebi.ac.uk:8053/ega-openid-connect-server"
- ;;                     :permission-server-url "https://ega.ebi.ac.uk:8053/ega-permissions"
- ;;                     :client-id "..."
- ;;                     :client-secret "..."}
+ ;; NB: currently not used as the only implementation was removed. Consider using and developing plugins.
  :entitlement-push nil
```

Comment:
Maybe. I thought that this would be the place where that config is put, but then again, the plugins can have their own config. I think I'll remove it entirely (and the reference to v1 v2)

Reply:
The reference is now removed and added comment about using plugins instead. I think there will be need to further develop the plugins until they are completely usable but it's the new future direction.